src/sidebar/components/annotation-share-control.js

```diff
@@ -141,14 +143,23 @@ function AnnotationShareControl({
                 className="annotation-share-panel__icon-button"
               />
             </div>
-            <div className="annotation-share-panel__permissions">
-              {annotationSharingInfo}
-            </div>
-            <ShareLinks
-              shareURI={shareUri}
-              analyticsEventName={analytics.events.ANNOTATION_SHARED}
-              className="annotation-share-control__links"
-            />
+            {inContextAvailable ? (
+              <div className="annotation-share-panel__details">
+                {annotationSharingInfo}
+              </div>
+            ) : (
+              <div className="annotation-share-panel__details">
+                This annotation cannot be shared in its original context because
+                it was made on a document that is not available on the web. This
+                link shares the annotation by itself.
+              </div>
+            )}
+            {inContextAvailable && (
+              <ShareLinks
+                shareURI={shareUri}
+                analyticsEventName={analytics.events.ANNOTATION_SHARED}
```

Comment:
This reminds me - perhaps we should just strip out all the "analytics" code for the time being since we're not using any of it.

Reply:
I'd be very +1 on that.